app/controllers/games.js

```diff
@@ -8,17 +8,93 @@ import Game from '../models/game';
  * @param {object} req
  * @param {object} res
  */
-const createGameLogs = (req, res) => {
-// save game if user is authenticated
+export const createGameLogs = (req, res) => {
+  // save game if user is authenticated
   if (req.decoded && req.params.id) {
     const game = new Game(req.body);
     game.userID = req.decoded._id;
     game.gameID = req.params.id;
     game.save((error) => {
-      if (error) return res.status(400).send({ message: 'Fail... game logs not saved' });
-      return res.status(201).send({ message: 'Success... game logs saved' });
+      if (error) {
+        return res.status(400).send({
+          message: 'Fail... game logs not saved'
+        });
+      }
+      return res.status(201).send({
+        message: 'Success... game logs saved'
+      });
     });
   }
 };
 
-export default createGameLogs;
+/**
+ * Sorted winners function
+ * @returns {Object} array showing the winners and the frequency of wins
+ * @param {Object} gameWinners
+ */
+const sortWinners = (gameWinners) => {
+  const sortedWinners = {};
+  gameWinners.forEach((index) => {
+    sortedWinners[index] = (sortedWinners[index] || 0) + 1;
+  });
+  return sortedWinners;
+};
+
+/**
+ * getGameWinners function
+ * @returns {Object} array of the individual game winners
+ * @param {Object} gameResults
+ */
+const getGameWinners = (gameResults) => {
+  const gameWinners = [];
+  gameResults.forEach((gameResult) => {
```

Comment:
Good job on this. 
Do you think it'll be great if we also get the points obtained by each player, so we can sort based on points, Or would you rather we sort based on wins?

Reply:
We are sorting based on wins, this is due to the fact that the game points are not logged in the game logs.